openapi/kas-fleet-manager.yaml

```diff
@@ -1114,6 +1123,9 @@ components:
         id:
           description: 'Unique identifier of this Kafka instance size.'
           type: string
+        display_name:
```

Comment:
Can we add these new fields to the [SupportedKafkaInstanceTypeListExample](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/dd5c689c8372d5e155cefcfeb47b53e7477d8028/openapi/kas-fleet-manager.yaml#L1698)?

Reply:
Added

Reply:
thanks!